src/components/pages/TodoHome.vue

```diff
@@ -49,4 +50,17 @@ function handleToggleButtons(id: string) {
 function isClicked(id: string) {
   return currentClickedId.value === id;
 }
+
+function moveChecked(id: string, isChecked: boolean) {
+  const index = todos.value.findIndex((todo) => todo.id === id);
+  if (index !== -1) {
```

Comment:
in cases like this its better to reverse the condition and use early return, so your code wont be indented a lot.